webrender/src/render_backend.rs

```diff
@@ -22,6 +22,7 @@ use resource_cache::ResourceCache;
 use scene::Scene;
 #[cfg(feature = "debugger")]
 use serde_json;
+use std::sync::atomic::{ATOMIC_USIZE_INIT, AtomicUsize, Ordering};
```

Comment:
The u32is not stable now. Use usize instead.

error: use of unstable library feature 'integer_atomics' (see issue #32976)
  --> src\render_backend.rs:25:25
   |
25 | use std::sync::atomic::{AtomicU32, Ordering};